src/components/TaxonDetails/TaxonDetails.js

```diff
@@ -122,6 +141,33 @@ const TaxonDetails = ( ): Node => {
             onPress={( ) => navigation.goBack( )}
           />
         </View>
+
+        <View className="absolute right-5 top-5">
+          <KebabMenu
+            visible={kebabMenuVisible}
+            setVisible={setKebabMenuVisible}
+            large
+            white
+          >
+            <Menu.Item
+              onPress={( ) => {
+                openURLInBrowser( taxonUrl );
+                setKebabMenuVisible( false );
+              }}
+              title={t( "View-in-browser" )}
+            />
+            <Menu.Item
+              onPress={( ) => {
+                Share.share( {
+                  message: taxonUrl,
```

Comment:
On iOS, having both message and url keys results in text that says "2 Links", as shown below. We solved this on ObsDetails using a [conditional statement](https://github.com/inaturalist/iNaturalistReactNative/blob/94ea2b6054dc06a0fea8e234e28a525ff08377ec/src/components/ObsDetails/HeaderKebabMenu.js#L26) for iOS vs. Android.

<img width="388" alt="Screenshot 2023-12-18 at 2 44 28 PM" src="https://github.com/inaturalist/iNaturalistReactNative/assets/35536439/7c22019e-ae11-4316-80bc-b11b9c1fe823">